packages/expo-brightness/src/Brightness.ts

```diff
@@ -52,6 +52,7 @@ export async function useSystemBrightnessAsync(): Promise<void> {
   if (Platform.OS !== 'android') {
     return;
   }
+  // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
Not sure what the right tradeoff is, but using just the `use` prefix to determine whether a method is a hook seems to limit naming for normal functions quite a bit. There are definitely valid cases where methods should be named in this pattern (`usePushCert`, `useMiddleware`). Maybe the heuristic the plugin uses will improve over time to eliminate the need for these exceptions. https://github.com/facebook/react/blob/master/packages/eslint-plugin-react-hooks/src/RulesOfHooks.js#L17

Don't think this should block adding the rule since the false positives seem to be in the minority, but might be worth thinking about how this could subconsciously influence developer behavior when naming functions.

Reply:
The hooks linter plugin is used a variety of apps even if they don't use our ESLint preset so I think we we should just accommodate it and rename this method eventually. I couldn't easily think of a better name for now and want to be thoughtful about it since it'd be a breaking change to our public API.